nodeJS/express/deployment.md

```diff
@@ -237,6 +237,12 @@ In most cases, you'll be running into errors that thousands of developers have e
 
 There are two stages of the deployment process where you are most likely to encounter problems. These are during deployment and right after.
 
+<div class="lesson-note" markdown="1">
+
+Depending on the host provider you use, the default NodeJs engine version might not be the LTS version. Additionally, depending on the NodeJs features you’re using, you might need to specify the NodeJs version to utilize those features.
```

Comment:
General suggestions for wording of the point, plus a nice link to how to actually specify compatible versions.

Plus, instead of "NodeJs" or "NodeJS", it should either be "Node.js" or just "Node" for consistency across documentation I think. I don't mind either and have used "Node" in the suggestion below. Whatever we end up using, would you mind also amending any other mentions of "NodeJS" in the lesson if there are any?

```suggestion
#### Node version compatibility

Depending on the host provider you use, the supported Node versions and default selected version may differ. You can refer to the providers' documentation for more information on what is supported and selected, and depending on what features you use in your code, you may need to [specify what Node versions your project is compatible with in your `package.json`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines).
```